### DISCUSSION TITLE
Your stars list  doesn't keep recently active setting

### DISCUSSION THREAD

**Discussion by StenApp** [community/community#62137] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi,

its painful to me to having always to change "Your stars" list recently starred to recently active.
I want to see what has changed recently.

**Comment by Cesar556** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> Hi,
> 
> its painful to me to having always to change "Your stars" list recently starred to recently active.
> I want to see what has changed recently.
> 

- **_[]()bug_**
